.idea/libraries/Gradle__android_arch_lifecycle_livedata_1_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
Add gitignore for those files. https://github.com/github/gitignore/blob/master/Android.gitignore

Reply:
Okay.